images.go

```diff
@@ -15,9 +17,9 @@ import (
 )
 
 const (
-	// ImageDigestBigDataKey is the name of the big data item whose
-	// contents we consider useful for computing a "digest" of the
-	// image, by which we can locate the image later.
+	// ImageDigestBigDataKey is a prefix of big data item names which we
+	// consider to be manifests, used for computing a "digest" value for
+	// the image as a whole, by which we can locate the image later.
 	ImageDigestBigDataKey = "manifest"
```

Comment:
Should the name of the variable be updated now that it is a prefix of a key?

Or, rather, if the goal is to store all of the manifests and to allow looking them up by digest, maybe c/storage should prescribe the manifest big data element naming, and provide an `ImageBigDataKeyForManifest(manifest []byte)` (or a `…ForDigest(digest digest.Digest)`?) so that this convention is defined in exactly one place.

Reply:
I wanted to avoid enforcing a convention so that the image library would be the place where that could be decided, but manifests are a concession to that.  One thing I'd been worried about was avoiding cases where a newer version of the image library that takes advantage of this was mixed with an older version of the storage library that wouldn't index the manifest's digest unless its name was exactly "manifest", and changing the name of the constant that an updated containers/image will use will accomplish that.  I'll leave the old name around for now so that we don't immediately break older versions of containers/image.